wiki/How_You_Can_Help!/en.md

```diff
@@ -1,77 +1,77 @@
-<!-- TODO: tons of things to add here. might be related to Projects article -->
+<!-- TODO: tons of things to add here. might be related to Projects articl -->
```

Comment:
One last thing here, I don't think this is intended.

Reply:
Yeah, my bad. I sometimes highlight stuff by accident